connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -494,15 +500,25 @@ private void expectTopicCreation(String topic) {
         }
     }
 
+    private void mockSinkTransform() {
+        FaultyPassthrough<SinkRecord> faultyPassthrough = new FaultyPassthrough<>();
+        @SuppressWarnings("unchecked")
+        Class<? extends Transformation<?>> value = (Class<? extends Transformation<?>>) (Class<?>) FaultyPassthrough.class;
+        OngoingStubbing<Class<? extends Transformation<?>>> transformClass = when(transformation.transformClass());
+        transformClass.thenReturn(value);
```

Comment:
```suggestion
        doReturn(FaultyPassthrough.class).when(transformation).transformClass();
```

This might be more readable, WDYT?

Reply:
This also works around the type issues i was having, incredible.